test/cmdLineTests/shareClassTests/DataHelperTests/playlist.xml

```diff
@@ -24,50 +24,18 @@
 
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TestConfig/playlist.xsd">
 	<test>
-		<testCaseName>cmdLineTester_DataHelperTests_win</testCaseName>
+		<testCaseName>cmdLineTester_DataHelperTests</testCaseName>
 		<variations>
 			<variation>Mode110</variation>
 			<variation>Mode610</variation>	
 		</variations>
-		<command>$(MKTREE) $(REPORTDIR); \
+		<command>$(MKDIR) -p $(REPORTDIR); \
```

Comment:
Should we still use `$(MKTREE) $(REPORTDIR);`?

Reply:
yes, updated.